Reportes/Minero Pineda Erick Rodrigo/reporteAndroid.md

```diff
@@ -0,0 +1,14 @@
+# Lo que aprendí en Android 📲
+## Erick Rodrigo Minero Pineda 👦🏽
+
+Comenzare por decir que por fin aprendí a usar GitHub, 😆 quizá aun no lo domino, pero, ahora me resulta sencillo utilizarlo para mis proyectos, previo al curso lo único que realizaba era subir el código mediante la pagina web y solo porque me lo pedían, en realidad no lo usaba. 💁🏽‍♂️
+
+Yo creo que es bastante útil, sobre todo ahora que tenemos que estudiar o trabajar desde casa, creo que me permite realizar el trabajo desde múltiples dispositivos, y además colaborar con compañeros. Ahora obtuve mi versión gratuita de PRO🌟, y estoy feliz por todo lo demás que incluye. Junto con esto descubrí que existe MarkDown, me parece importante porque es una forma bastante sencilla de documentar en GitHub, no solo eso, también es importante porque puede ser utilizado en Notebooks de Jupyter, o de MATLAB,📗 se pueden armar documentos bastante decentes y estéticamente bonitos. También es sencillo de utilizar ya que es muy similar a utilizar un lenguaje de etiquetas como HTML, o de alguna manera se puede decir que se parece a LaTex.
+En lo personal para escribir en MD me gusto Atom, me gusta su simplicidad, y que podemos ver el contenido en tiempo real. Además, tengo un gusto por ese editor de texto.
+
+Por otro lado, desarrollar aplicaciones con Android no me era desconocido 🛠, ya que tenia ligeros conocimientos acerca de Android con Java, sin embargo, Kotlin para mi es nuevo, o al menos lo era, ahora considero que Kotlin es un poco mas robusto que Java para el desarrollo de aplicaciones para Android, me parece que no solo para eso, pues como lo vimos en el curso es mas sencillo en algunos aspectos como lenguaje normal de programación. No considere complicado entenderlo ya que como esta basado en Java su sintaxis es muy similar.
+Ahora quede bastante picado en conocer y mejorar mi dominio en Kotlin, ya que java pues, es viejito y creo que lo nuevo será Kotlin, por lo que buscare mejorar en ello, además que me gusta desarrollar mis propias aplicaciones.🪁
+
+En cuanto a el desarrollo como tal 💻, me gusta Android Studio, no tuve ningún problema para su instalación o durante el uso del programa,🙏🏻 yo le agregaría más formas de personalización, es decir temas, o cosas así. Algo que me gusto mucho es que es muy sencillo vincularlo con GitHub, y realizar commits o push. Me resulto muy sencillo el despliegue de ítems o Widgets como les llama Google, la verdad no me agrada la vista de código, prefiero realizarlo todo mediante el inspector de propiedades,🔍 ya que no logro aprenderme todos los nombres de los atributos, y en el inspector me es más fácil ubicarlos.🕶
```

Comment:
Como dijo un profesor de la Unam: Lástima Margarito porque creo que a nivel top industria con puro código de Kotlin se echan las interfaces tipo this.botón.layout.constraintLeft, muy poco o no tanto de XML, para evitar que la clase R se ponga medio fresa y no acepte las uniones.